### PR TITLE
integrate latex

### PR DESCRIPTION
**Pull request by minhrongcon2000**
This pull request proposes an integration of latex to the theme! I believe latex is crucial for bloggers who want to write academic blog!

### CODE REVIEW

**Comment by thelehhman** [comment]
Thank you @minhrongcon2000 , this looks great! Couple of points though -
- I don't think we need jQuery to use MathJax, so that can be removed
- We should make "latex" an on-demand feature, i.e. only load the relevant scripts if they are specified. For example, search(https://github.com/thelehhman/plainwhite-jekyll/pull/37) and dark mode(https://github.com/thelehhman/plainwhite-jekyll/pull/39) are on-demand features
- The example is great, but any relevant documentation to enable and use this option should be added to the README.

**Comment by thelehhman** [comment]
Cool

**Comment by minhrongcon2000** [comment]
@thelehhman I have added the usage to README, and also change the latex to on-demand feature. It will only be loaded if the user set latex field in `_config.yml` to true, and add math_engine to kramdown. Could you have a look on that?

File: _config.yml [hunk 1]
```diff
@@ -48,6 +49,9 @@ plainwhite:
 
 # Build settings
 theme: plainwhite
+markdown: kramdown
+kramdown:
+  math_engine: mathjax
```
Comment (thelehhman) [inline _config.yml:54]: Here too

**Comment by samarsault** [comment]
Hey, @minhrongcon2000 are you still interested in pursuing this?

**Comment by minhrongcon2000** [comment]
Sorry @samarsault ! I'm now on another project! I would close this for now! Sorry for my inconvenience!

**Pull request closed without merging**